tests/test_component.py

```diff
@@ -85,3 +85,13 @@ def test_has_component_1(self) -> None:
         self.assertEqual(len(bom.components), 2)
         self.assertTrue(bom.has_component(component=get_component_setuptools_simple_no_version()))
         self.assertIsNot(get_component_setuptools_simple(), get_component_setuptools_simple_no_version())
+
+    def test_get_component_by_purl(self) -> None:
```

Comment:
:x:  adde to wrong file.
this is a test on `Bom` model., not `Component`

Reply:
no problem, but then the test above it `test_has_component_1()` should also be moved?!

Reply:
you are right. could you carry this one over to the other file, too?

Reply:
sure thing 😄